react-native.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  dependency: {
```

Comment:
Why do you add `dependency` here instead using `project`?

Reply:
Since RNCP is a [library](https://github.com/react-native-community/cli/blob/master/docs/configuration.md#libraries) not an app.

Reply:
oh right, thanks for your information